### ISSUE TITLE
Blog post about seven tips for improving accessibility of Quarto documents

### ISSUE THREAD

**Issue by remlapmot** [mcanouil/awesome-quarto#421]
### Link to suggestion

https://remlapmot.github.io/post/2025/quarto-accessibility/

### Targeted section

Blog posts

### Describe your suggestion

I've written a blogpost about seven top tips for Quarto and R Markdown users on document accessibility, including; document format, adding alt text, creating tagged pdfs, table accessibility, different accessibility checkers, custom Word document templates, and how to upload complex html documents into Blackboard Ultra.

### Checklist

- [x] checked if your suggestion is not already listed in __Awesome Quarto__ list.
- [x] checked if it can fit in an existed section or suggest a new section and explain why your item does not fit into one and what would be the interest of your new section.
- [x] checked when it's a repository if it's maintained (archived repository will be rejected).

**Comment by mcanouil** [comment]
I cannot add all your blog posts to the list even though they are interesting.
(I did not add all of mine for instance.)
